crates/gourgeist/src/bare.rs

```diff
@@ -174,9 +156,19 @@ pub fn create_bare_venv(
     }
 
     // Add all the activate scripts for different shells
-    // TODO(konstin): RELATIVE_SITE_PACKAGES is currently only the unix path. We should ensure that all launchers work
-    // cross-platform.
     for (name, template) in ACTIVATE_TEMPLATES {
+        let relative_site_packages = if cfg!(unix) {
```

Comment:
I'm wondering whether using conditional compilation for logic like this is the right thing to do. For example, are we sure that all `uv` binaries compiled for Windows will 100% of the time use the Windows oriented path below? Or is it possible for the Unix path (WSL?) to be used in a Windows context?

Reply:
Same deal for use of conditional compilation elsewhere.

Reply:
Yeah probably not, but I think it's ~the same as it is today. I have an idea for how to fix this properly (#2095).

Reply:
Unfortunately they don't (https://github.com/astral-sh/uv/issues/1251), but #2095 should solve either case